### PR TITLE
Show user, Statistics in Admin #2550, Skill Example #2635

### LINKED ISSUE #2550: Admin Dashboard: Show User and Skill Statistics
The information on user activities is limited on the admin dashboard.

Please implement a chat for user information with the following:
* Logins over time
* User registration over time
* Skill edits over time
* Skill creations over time
* Any other interesting information

Screenshot from 2019-07-14 18-11-28

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #2550, #2635

Changes:
- [x] Fix show more example
- [x] Skill edits over time
- [x] Skill creations over time
- [x] Skill Uses over Time
- [x] Logins over time
- [x] User registration over time

Demo Link: https://pr-2636-fossasia-susi-web-chat.surge.sh

Screenshots of the change: 
Screenshot 2019-07-19 at 2 43 02 AM

Screenshot 2019-07-19 at 2 43 07 AM

Screenshot 2019-07-24 at 10 18 36 PM

### CODE REVIEW

**Comment by fragm3** [comment]
Note: I have added the required code for all the 5 types of chat data over time, just need the backend to work fine now. I have asked @Orbiter for help in this

File: src/components/Admin/AdminTab/AdminTab.js [hunk 1]
```diff
@@ -111,6 +147,43 @@ class AdminTab extends React.Component {
...
+      obj.timeStamp = moment(data.timeStamp).format('MM/YY');
+      obj.count = data.count;
+      return obj;
+    });
+    return (
+      data.length > 0 && (
+        <Card height={'400'} width={'90%'} margin={'0 10px'}>
+          <CardHeading>{heading}</CardHeading>
+          <ResponsiveContainer height={300} width={'100%'} debounce={1}>
+            <LineChart data={chartData}>
```
Comment (akshatnitd) [inline src/components/Admin/AdminTab/AdminTab.js]: Make a shared component for it. Put it in `shared/charts/LineChart.js` path.

File: src/components/Admin/AdminTab/AdminTab.js [hunk 2]
```diff
@@ -243,6 +320,36 @@ class AdminTab extends React.Component {
                   </CardContentContainer>
                 </Card>
               </CardContainer>
+              {lastModifiedOverTime.length > 0 &&
```
Comment (akshatnitd) [inline src/components/Admin/AdminTab/AdminTab.js]: Make a config for it and render by mapping over it.

File: src/components/cms/SkillPage/SkillListing.js [hunk 3]
```diff
@@ -307,15 +309,15 @@ class SkillListing extends Component {
     let renderElement = null;
     if (examples.length > 4) {
       seeMoreSkillExamples = seeMoreSkillExamples ? (
-        <MoreExamplesContainer>
+        <MoreExamplesButton variant="outlined" color="primary">
```
Comment (akshatnitd) [inline src/components/cms/SkillPage/SkillListing.js]: Keep the UI such that, the button remains inline with the example.

File: src/components/Admin/AdminTab/AdminTab.js [hunk 4]
```diff
@@ -111,6 +147,43 @@ class AdminTab extends React.Component {
       });
   }
 
+  renderChart = (data, heading, legend) => {
+    const dateSortedData = sortByDate(data);
+    const chartData = dateSortedData.map(data => {
+      const obj = {};
```
Comment (akshatnitd) [inline src/components/Admin/AdminTab/AdminTab.js]: `obj` -> `chartObj`

**Comment by fragm3** [comment]
@akshatnitd made the changes:
Screenshot 2019-07-25 at 2 37 19 AM
Screenshot 2019-07-25 at 2 37 27 AM

**Pull request closed without merging**